core/rawdb/accessors_chain_test.go

```diff
@@ -215,6 +219,99 @@ func TestPartialBlockStorage(t *testing.T) {
 	}
 }
 
+func TestTransactionStorage(t *testing.T) {
```

Comment:
from simple (one goroutine) test point of view - do commit or not - doesn't change anything. 
It means can do: 
```
_, tx := kv.NewTestTx(t)
... do reads and writes on 1 tx
t.Cleanup will rollback tx and close db automatically after all sub-tests finish
```